Source/Riemann.H

```diff
@@ -171,4 +171,94 @@ riemann(
   uflx_eint = qint_iu * regd;
 }
 
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+laxfriedrich_flux(
+  const amrex::Real rl,
+  const amrex::Real ul,
+  const amrex::Real vl,
+  const amrex::Real v2l,
+  const amrex::Real pl,
+  const amrex::Real spl[NUM_SPECIES],
+  const amrex::Real rr,
+  const amrex::Real ur,
+  const amrex::Real vr,
+  const amrex::Real v2r,
+  const amrex::Real pr,
+  const amrex::Real spr[NUM_SPECIES],
+  const int,
+  const amrex::Real,
+  amrex::Real& ustar,
+  amrex::Real& uflx_rho,
+  amrex::Real& uflx_u,
+  amrex::Real& uflx_v,
+  amrex::Real& uflx_w,
+  amrex::Real& uflx_eden,
+  amrex::Real& uflx_eint,
+  amrex::Real&,
```

Comment:
Why not just delete these unused arguments all together?

Reply:
actually I agree. I can do this if @marchdf agrees that this is not a Riemann solver. I was just trying to be consistent with the riemann function call.

Reply:
fine by me!

Reply:
made this change now!